recipes/ml4xcube/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "ml4xcube" %}
+{% set version = "0.0.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ml4xcube-{{ version }}.tar.gz
+  sha256: 3f58ed6e4babf54cf935bc881348ceea99c8207c24e878d2e793bb8bb9221903
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+  run:
+    - python >=3.8
+    - conda-forge::xcube
```

Comment:
I'm not familiar with the `::` in a package name preceded by the the channel (?). I think this could just be `xcube`. Could you clarify why the dependency is listed the way that it is?

Reply:
Hi @synapticarbors 

Thank you for reviewing my Python package!

I wanted to install `xcube` specifically from the conda-forge channel because the latest versions of this package are only available there. 

As mentioned in [this GitHub issue](https://github.com/conda/conda/issues/7202), the syntax `channel::package` can be used in an `environment.yml` file to force the installation from a specific channel. I assumed this would work in the `meta.yaml` as well, so I used this approach to ensure that the package is installed from conda-forge.

Since I am not shure if this is the right approach for `meta.yaml` I simply added a package version to the definition.

Reply:
On conda-forge we only pull from the conda-forge channel for requirements, and for end-users it's probably better to specify the correct version rather than trying to force a particular channel.

Reply:
Hi @synapticarbors,

Thank you for merging my code. I have been trying to find the ml4xcube package in the available conda-forge packages, but it seems it is not yet available for download. Could you please let me know when it will be possible to download it?

As I am new to this procedure, I would appreciate your feedback in this regard.

Best,
Julia

Reply:
In the commit history I saw that the receipe was removed. Do I need to make another PR?

![grafik](https://github.com/conda-forge/staged-recipes/assets/51787615/0bc49d63-efc9-403d-b549-9961b457f008)

Reply:
The feedstock was created here https://github.com/conda-forge/ml4xcube-feedstock but it didn't actual build the packages for some reason. Let me look into it.

Reply:
Hi @synapticarbors,

I hope you're doing well. I wanted to follow up on the status of the ml4xcube package. Is there any update on why the package build didn't succeed. How could I proceed from here?

Reply:
See https://github.com/conda-forge/ml4xcube-feedstock/pull/2#issuecomment-2118389200 -- one way to proceed would be to remove the `pip check` there which I can do